fli-iam/shanoir-ng
# shanoir-issue#2155: improve delete of examination

Fix for #2155 

How to test:
Create a study
Import datasets in it
Delete the examination that contains the imported dataset

This fix should reduce the duration of exam deletion


Also fixes:
Deletion of copied datasets / datasets acquisitions / examinations
Deletion of subjects
Add subject's name in dataset acquisition list
